tests/integration/crypto_tests.cpp

```diff
@@ -64,4 +64,194 @@ BOOST_FIXTURE_TEST_CASE( keccak_tests, tester ) try {
 
 } FC_LOG_AND_RETHROW()
 
-BOOST_AUTO_TEST_SUITE_END()
\ No newline at end of file
+BOOST_FIXTURE_TEST_CASE( bn128_add_tests, tester ) try {
+   create_accounts( { "test"_n } );
+   produce_block();
+
+   set_code( "eosio"_n,  contracts::crypto_primitives_test_wasm() );
+   set_abi(  "eosio"_n,  contracts::crypto_primitives_test_abi().data() );
```

Comment:
I suggest to make tester wrapper class to not duplicate `create_accounts` and `set_code/set_abi` in every single test. Test in that case will be declared like that:
`BOOST_FIXTURE_TEST_CASE( bn128_add_tests, crypto_primitive_tester )`

Reply:
Thanks! Good point. Changed.